libparsec/crates/client/src/user_ops/mod.rs

```diff
@@ -82,14 +82,15 @@ impl UserOps {
         message::process_last_messages(self, latest_known_index).await
     }
 
-    pub async fn sync(&self) -> Result<(), SyncError> {
-        sync::sync(self).await
-    }
-
     /*
      * Public interface
      */
 
+    // TODO: It has been moved in the public interface, because some of the commands in the CLI didn't sync automatically
```

Comment:
What should be done ? `sync` should not be publicly exposed ?

Reply:
I don't know but I have changed something, that's why I mention it

Reply:
It's better to know no ?

Reply:
According to @touilleMan it's something low level that we don't want to expose, but it should be fixed if we want to keep private.

Reply:
Then can you update the comment to state that the method is currently expose but it's not something we want ?

Reply:
Sure

Reply:
Done